boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

```diff
@@ -270,6 +272,11 @@ You should see the following message on the console:
 
    Hello World! arm
 
+Building a secure/non-secure with Arm |reg| TrustZone |reg|
+-----------------------------------------------------------
+
+The TF-M integration sample :ref:`tfm_ipc` can be run by a Nucleo L552ZE Q, using the ``nucleo_l552ze_q_non_secure`` target.
```

Comment:
After some recent conversation on the topic, we've decided to settle on `_ns` for the nonsecure suffix. As such, please update the documentation and code to use the more concise `nucleo_l552ze_q_ns` (sorry, this was a later decision, but consistency is important on these things).